fix(docs): tutorials tags cleanup final batch (#3318)
scaleway/docs-content: tutorials/setup-lamp-stack-ubuntu2204/index.mdx

    ```
    Switch to unix_socket authentication [Y/n]
    ```
4. Press `N` to keep your current root password, then press `Y` to change the root password:
    ```bash
    You already have your root account protected, so you can safely answer 'n'.

    Change the root password? [Y/n]
    ```
5. Enter the new password if you decide to change the root password, press `Enter`, then repeat this step.

> 6. Keep the settings proposed by default when prompted by further questions, by pressing `Enter` each time.

Reviewer: ```suggestion
6. Press `Enter` to keep the settings proposed by default when prompted.
```
